### PR TITLE
Clarified Rancher URL var and pre-req

### PR DESCRIPTION
**Pull request by oleg-vorobiov-suse**
Rancher URL var has to be in a certain format
rancher-charts helm repo has to be added to a machine where this script is running from

### CODE REVIEW

File: adjust-downstream-webhook/README.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ Since each version of the webhook is one-to-one compatible with a specific versi
 ```bash
 ## Create a token through the UI. The token should have no scope and be made for a user who is a global admin.
 read -s RANCHER_TOKEN && export RANCHER_TOKEN
-## The server URL for Rancher - you can get this value in the "server-url" setting.
+## The server URL for Rancher - you can get this value in the "server-url" setting. The example format should be: https://rancher-test.home
```
Comment (JeffParkes) [inline adjust-downstream-webhook/README.md:15]: Maybe mention that you can find this in in Rancher global settings > settings > server-url.

Comment (dkeightley) [reply adjust-downstream-webhook/README.md:15]: @oleg-vorobiov-suse would you be able to add this? If you agree, looks like we can merge once added

Comment (oleg-vorobiov-suse) [reply adjust-downstream-webhook/README.md]: Just added Jeff's changes, I am good

**Review by JeffParkes — APPROVED** [review]
LGTM

**Review by dkeightley — APPROVED** [review]

**Pull request merged**